local-cli/runAndroid/runAndroid.js

```diff
@@ -123,18 +123,19 @@ function buildAndRun(args, reject) {
 }
 
 function startServerInNewWindow() {
+  var yargV = require('yargs').argv;
+
   const launchPackagerScript = path.resolve(
     __dirname, '..', '..', 'packager', 'launchPackager.command'
   );
 
   if (process.platform === 'darwin') {
-    child_process.spawnSync('open', [launchPackagerScript]);
+    const termApp = yargV.open ? yargV.open : 'Terminal.app';
+    return child_process.spawnSync('open', ['-a', termApp, launchPackagerScript]);
```

Comment:
That is the way it was but people made comments to change it to this. So who the heck do I follow?